test/mode/modeVisualBlock.test.ts

```diff
@@ -188,4 +188,12 @@ suite('Mode Visual Block', () => {
     keysPressed: '<C-v>llj"ayGo<C-r>a<Esc>',
     end: ['abcde', '01234', 'abcde', '01234', '123', 'bcd', '|'],
   });
+
+  newTest({
```

Comment:
Can you write a test where the entire visual block is on the same line? It's a bit of a special case.